yarn-project/acir-simulator/src/client/private_execution.test.ts

```diff
@@ -372,33 +369,6 @@ describe('Private Execution test suite', () => {
       expect(recipientNote.preimage[0]).toEqual(new Fr(amountToTransfer));
       expect(changeNote.preimage[0]).toEqual(new Fr(balance - amountToTransfer));
     });
-
-    it('Should be able to claim a note by providing the correct secret and nonce', async () => {
```

Comment:
This test is no longer needed since it's the same as transfer (reading and removing note from a set).